frontend/app/components/new-tab-indicator.tsx

```diff
@@ -0,0 +1,10 @@
+import type { ComponentProps, ReactNode } from 'react';
+
+export interface NewTabIndicatorProps extends ComponentProps<'span'> {
+  children: ReactNode;
+}
```

Comment:
Omit children prop since we will handle the content in the component.

```suggestion
export interface NewTabIndicatorProps extends Omit<ComponentProps<'span'>, 'children'> { }
```